examples/cpp/core/capped_allocator/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_bond_codegen (capped_allocator.bond
+    OPTIONS 
+        --allocator=\"bond::capped_allocator<std::allocator<void> >\"
```

Comment:
Should we make `std::allocator` the default?

Reply:
Yeah, I think that makes sense.